packages/now-build-utils/src/detect-routes.ts

```diff
@@ -83,7 +83,7 @@ function createRouteFromPath(filePath: string): Route {
 
   const dest = `/${filePath}${query.length ? '?' : ''}${query.join('&')}`;
 
-  return { src, dest };
+  return { src, dest, check: true };
```

Comment:
@dav-is  Should the the `enableExperimentalHandleMiss` flag also affect the `check: true` property?